packages/core/src/shader/utils/checkMaxSamplersInShader.js

```diff
@@ -0,0 +1,20 @@
+export function checkMaxSamplersInShader(maxSamplers, gl)
+{
+    if (maxSamplers === 0)
+    {
+        throw new Error('Invalid value of `0` passed to `checkMaxSamplersInShader`');
+    }
+
```

Comment:
hey ya, 'debugInfo' could be null on this check. This would then throw an error.

https://developer.mozilla.org/en-US/docs/Web/API/WEBGL_debug_renderer_info

if you could cater for that, then this PR will be good to go 👍 

Cheers!